plugins/inputs/wireless/wireless.go

```diff
@@ -7,19 +7,23 @@ import (
 
 // Wireless is used to store configuration values.
 type Wireless struct {
-	HostProc string          `toml:"host_proc"`
-	Log      telegraf.Logger `toml:"-"`
+	HostProc  string          `toml:"host_proc"`
+	Log       telegraf.Logger `toml:"-"`
+	DumpZeros bool            `toml:"dump_zeros"`
 }
 
 var sampleConfig = `
-  ## Sets 'proc' directory path
-  ## If not specified, then default is /proc
-  # host_proc = "/proc"
-`
+[wireless]
+   ## Sets 'proc' directory path
+   ## If not specified, then default is /proc
+   # host_proc = "/proc"
+	 ## dump metrics with 0 values too
+	 # dump_zeros = true
+ `
 
 // Description returns information about the plugin.
 func (w *Wireless) Description() string {
-	return "Monitor wifi signal strength and quality"
+	return "Monitor wifi signal strength and quality on macOS"
```

Comment:
Please don't change this as it also works for Linux.

Reply:
reverted. Shouldn't have been there anyway